inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp

```diff
@@ -66,17 +66,32 @@ class MVNStage final : public StageNode {
 
 }  // namespace
 
-void FrontEnd::parseMVN(const Model& model, const ie::CNNLayerPtr& _layer, const DataVector& inputs, const DataVector& outputs) const {
-    IE_ASSERT(inputs.size() == 1);
+void FrontEnd::parseMVN(const Model& model, const ie::CNNLayerPtr& layer, const DataVector& inputs, const DataVector& outputs) const {
```

Comment:
We cannot support IRs with old MVN, right? Or new "MVN1->MVN6" transformation is already enabled?

Reply:
MO has not supported MVN1 already. ngraph has MVN6->MVN1 temporary transformation that will be removed when all plugins will support MVN6

Reply:
MVN1->MVN6 transformation looks enabled, but it still not work, so deprecated tests failed and old MVN support as well. I have tried to enable this transformation manually in frontend, but it doesn't take result as well. I researching it now. Also I have tried to adapt deprecated test IR with pointing of opset, it doesn't take result.

Reply:
@andrei-kochin, please, confirm

Reply:
> MO has not supported MVN1 already. ngraph has MVN6->MVN1 temporary transformation that will be removed when all plugins will support MVN6

There is not MVN6->MVN1 transformation as far as I understand. There is decomposition from MVN6 to many atomic operations.